source/agora/consensus/protocol/Nominator.d

```diff
@@ -371,9 +376,8 @@ extern(D):
     {
         scope (failure) assert(0);
 
-        if (slot_idx in this.externalized_slots)
+        if (slot_idx <= this.ledger.getBlockHeight())
```

Comment:
`slot_idx` is the same as `block_height`. Doesn't slot_idx start with 0?

Reply:
`slot_idx` is set here: https://github.com/bpfkorea/agora/blob/7f52a574e07366b4c468f96f7deff207249ac01f/source/agora/consensus/protocol/Nominator.d#L167

Reply:
Yeah, I got it, thank you.